lib/logstash/outputs/elasticsearch.rb

```diff
@@ -128,6 +128,11 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # a timeout occurs, the request will be retried.
   config :timeout, :validate => :number
 
+  # Set the Elasticsearch errors that you don't want to log. By default we log 
+  # every error. A useful example is when you want to skip all 409 errors
+  # which are `document_already_exists_exception`
+  config :skip_errors, :validate => :array, :default  => [""]
```

Comment:
This should probably use `:default => []` right? (why have an array default to containing 1 element of an empty string?)